packer_images/ccd_logstash.conf.in

```diff
@@ -59,61 +59,77 @@ filter{
+            }
+        }
+        if [data][caseManagementCategory] {
+            mutate {
+               rename => {"[data][caseManagementCategory]" => "[data_new][caseManagementCategory]" }
+            }
         }
         if [supplementary_data][HMCTSServiceId] {
-           mutate {
-              rename => {"[supplementary_data][HMCTSServiceId]" => "HMCTSServiceId" }
-           }
+            mutate {
+               rename => {"[supplementary_data][HMCTSServiceId]" => "[supplementary_data_new][HMCTSServiceId]" }
+            }
         }
-
-        if [data][caseNameHmctsInternal] {
+        if [data_classification][SearchCriteria] {
+            mutate {
+               rename => {"[data_classification][SearchCriteria]" => "[data_classification_new][SearchCriteria]" }
+            }
+        }
+        if [data_classification][caseManagementLocation] {
            mutate {
-              rename => {"[data][caseNameHmctsInternal]" => "caseNameHmctsInternal" }
+              rename => {"[data_classification][caseManagementLocation]" => "[data_classification_new][caseManagementLocation]" }
            }
         }
-
-        if [data][caseManagementLocation][region] {
+        if [data_classification][caseNameHmctsInternal] {
             mutate {
-               rename => {"[data][caseManagementLocation][region]" => "region" }
+               rename => {"[data_classification][caseNameHmctsInternal]" => "[data_classification_new][caseNameHmctsInternal]" }
             }
         }
 
-        if [data][caseManagementLocation][baseLocation] {
+        if [data_classification][caseManagementCategory] {
             mutate {
-              rename => {"[data][caseManagementLocation][baseLocation]" => "baseLocation" }
+               rename => {"[data_classification][caseManagementCategory]" => "[data_classification_new][caseManagementCategory]" }
             }
         }
+        mutate { remove_field =>[ "data" ,"supplementary_data", "data_classification" ] }
+
         mutate {
-            rename => { "[reference]" => "caseReference"}
-            rename => { "[jurisdiction]"  => "jurisdictionId"}
-            rename => { "[state]"  => "stateId"}
-            rename => { "[case_type_id]"  => "caseTypeId"}
+                rename => { "[data_new]" => "data" }
+                rename => { "[supplementary_data_new]"  => "supplementary_data" }
+                rename => { "[data_classification_new]"  => "data_classification" }
         }
+
         mutate {
            add_field => { "index_id" => "global_search" }
         }
         mutate {
            lowercase => [ "index_id" ]
         }
 
-       mutate { remove_field => [ "last_state_modified_date", "type","last_modified", "created_date", "security_classification","json_data_classification", "data_classification", "json_supplementary_data", "supplementary_data", "data" ]}
+       mutate { remove_field => [ "last_state_modified_date", "type","last_modified", "created_date" ]}
```

Comment:
Minor: we could just combine these field removals into L109?

Reply:
Combined two lines